### DISCUSSION TITLE
Scipy

### DISCUSSION THREAD

**Discussion by dineshbvadhia** [python/typing#1310] [Q&A]
`import scipy`

causes the error: Skipping analyzing "scipy": module is installed, but missing library stubs or py.typed marker  [import]

Adding a `# type: ignore` works.

The scipy code includes a mypy.ini file - (https://github.com/scipy/scipy/blob/main/mypy.ini) - with the line:
`plugins = numpy.typing.mypy_plugin`

If scipy uses the numpy mypy plugin then why is the `# type:ignore` needed?

**Comment by jorenham** [comment]
The error will go away if you use [`scipy-stubs`](https://github.com/jorenham/scipy-stubs).
